coalib/bears/Bear.py

```diff
@@ -155,6 +156,13 @@ def can_detect(cls):
         return cls.CAN_DETECT | cls.CAN_FIX
 
     @classproperty
+    def file_path(cls):
+        """
+        :return: The file where the bear was fetched from.
+        """
+        return inspect.getfile(cls)
```

Comment:
Since all bears are defined in python source files, both will point to the same path.